packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -3452,102 +3488,126 @@ class EditableTextState extends State<EditableText> with AutomaticKeepAliveClien
     TransposeCharactersIntent: _makeOverridable(_transposeCharactersAction),
   };
 
+  // The default behavior if widget.onTapOutside is null.
+  void _handleDefaultTapOutside() {
+    /// The focus dropping behavior is only present on desktop platforms
+    /// and mobile browsers.
```

Comment:
Maybe this PR is not the place to worry about this, but does the behavior change for mobile devices with mice or desktop devices with touchscreens?  Relevant issue: https://github.com/flutter/flutter/issues/106586